ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.15-lp-interop.yaml

```diff
@@ -61,7 +61,7 @@ tests:
           {"step": "cluster-install", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","mtc-lp"]},
           {"step": "mtc-prepare-clusters", "failure_type": "all", "classification": "Infrastructure Deprovisioning - Cluster Configuration", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","mtc-lp","interop-tests"]},
           {"step": "cluster-uninstall", "failure_type": "all", "classification": "Infrastructure Deprovisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["4.15-lp","self-managed-lp","mtc-lp"]},
-          {"step": "mtc-execute-tests", "failure_type": "pod_failure", "classification": "Test Execution", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","mtc-lp","interop-tests"]},
+          {"step": "mtc-execute-tests", "failure_type": "pod_failure", "classification": "Test Execution", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","mtc-lp","interop-tests"]},
```

Comment:
I'm not sure if we should add this, this step is separate from any cluster specific step, this is just executing the PQE test suite. @calebevans thoughts?

Reply:
Yeah this doesn't have anything to do with the cluster, it is just the test suite. I guess they would fail if the cluster doesn't provision correctly though.

Reply:
I think it's a mistake (deleted)
If this was considered, all test pods should have been grouped as cluster with prioritize 2